.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+services:
+  app:
+    build:
+      context: ..
+      dockerfile: .devcontainer/Dockerfile
+    volumes:
+      - ../..:/workspaces:cached
```

Comment:
I wonder a bit why they mount the parent directory of the project in the [template](https://github.com/devcontainers/templates/blob/main/src/postgres/.devcontainer/devcontainer.json), but I kept it that way.